gradle/dependencies.gradle

```diff
@@ -42,9 +41,16 @@ configurations.all {
 }
 
 dependencies {
+    implementation('com.nike:cerberus-client'){
+        version {
+            branch = 'master'
+        }
+    }
+    compile "com.amazonaws:aws-java-sdk-core:${AWS_SDK_VERSION}"
```

Comment:
Why did this get added?

Reply:
This dependency got added because with the upgrade from Java 7 to Java 8, it needed to be referenced here.

It is used for AWS regions [here](https://github.com/Nike-Inc/cerberus-archaius-client/pull/67/files#diff-3fd683d9f3a2dbe81388d3df944a469ab395ce33411e07a56958abeb0f1288abR19).